frontend/src/components/login/LoginForm.tsx

```diff
@@ -16,17 +21,32 @@ import {SetJWTAsCookie} from "@/utils/oauth";
  */
 export default function LoginForm() {
     const router = useRouter();
+    const searchParams = useSearchParams()
 
     const [isOpen, setIsOpen] = useState(false);
+    const [error, setError] = useState<Error>(null);
```

Comment:
TS2345: Argument of type null is not assignable to parameter of type Error | (() => Error)